libs/coin-evm/src/editTransaction/index.ts

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file: pure exports */
```

Comment:
Why is it needed ?

Reply:
To explicitly skip the coverage requirement for this file, since this is an empty shell only used to export functions that are already unit-tested in their modules.

Similar to what is done in other pure exports files, like https://github.com/LedgerHQ/ledger-live/blob/develop/libs/coin-evm/src/api/node/rpc.native.ts

cf. https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md